docs/tips.md

```diff
@@ -1,34 +1,34 @@
 ## Training/test Tips
```

Comment:
This comes mainly from the cyclegan repo, let's update it completely, and rewrite accordingly where needed. Typically cyclegan is not the algorithm used by default in joligan anymore.